ONSdigital/ssdc-rm-ddl
# 2884 Add RM UI group setup scripts

# Motivation and Context
We need some groups defined for day to day RM support and elevated write access roles for the support tool UI/r-ops.

# What has changed
- Add UI permissions scripts

# How to test?
Check the lists of permissions make sense, RM SUPPORT should be all the global read only permissions and RM ELEVATED should have write access but only on our non prototype features.

# Links
https://trello.com/c/ZxqCN8wb/2884-productionise-support-tool-set-up-groups-3
